samples/SwiftDemoApp/SwiftDemoApp/Samples.swift

```diff
@@ -13,11 +13,11 @@
  * limitations under the License.
  */
 
-import Foundation
+import UIKit
 
 class Samples {
-  class func loadSamples() -> [Any] {
-    return [ newDemo(controller: ClusteringViewController.self,
+    class func loadSamples() -> [[String: Any]] {
```

Comment:
Since you are refactoring this class and adding a more specific type, we should also update this from a dictionary with arbitrary keys to a more specific type. A Swift enum would be better semantics:

```swift
enum Sample: CaseIterable {
    case Clustering
    case KML
    // ...other cases
}

extension Sample {
    var title: String {
        switch self { 
            case .Clustering: return "Clustering"
            // ...other cases
        }
    }

    // ...other properties
}
```

Conformance to `CaseIterable` would also replace `Samples.loadSamples()` to simply `Sample.allCases`.

Reply:
Actually thought like this. Yes, will do add it tomorrow and update the pr.  Thanks @arriolac 👍

Reply:
@arriolac done 👍

Reply:
Sound's Good.